File: src/main/kotlin/FizzBuzz.kt
class FizzBuzz {
    fun fizzBuzz(n: Int): Unit {
        // Write your code here
        for(i in 1..n){
            if(i%15 == 0){
                print("FizzBuzz \n")
            }else if(i%5 == 0){
                print("Buzz \n")
            }else if(i%3 == 0){
                print("Fizz \n")
            }else{
                print("$i \n")
            }
        }

    }
    fun main(args: Array<String>) {
        val n = readLine()!!.trim().toInt()
        fizzBuzz(n)
    }
}

File: src/main/python/algo/fizzbuzz.py

def fizzBuzz(n):
    for i in range(n):
        if i % 3 == 0:
            print("Fizz")
        if i % 5 == 0:
            print("Buzz")
        if i % 15 == 0:
            print("Fizz Buzz")
        else:
            print(i)


for i in range(100):
        fizzBuzz(i)